### PR TITLE
SeedImage: build image for the builder pod

### PR DESCRIPTION
**Pull request by fgiudici**
This allows to create a container image to be used for the Elemental seed image building.

Fixes https://github.com/rancher/elemental-operator/issues/374

### CODE REVIEW

**Review by davidcassany — APPROVED** [review]
LGTM

**Review by davidcassany — CHANGES_REQUESTED** [review]
Wait, just saw the obs chart is not updated

**Comment by davidcassany** [comment]
I do believe this will be inconsistent with the OBS chart. I'd say the seed image value should be included within `.obs/chartfile/values.yaml`

File: .obs/chartfile/values.yaml
Comment (fgiudici) [inline .obs/chartfile/values.yaml:1]: Hey @davidcassany , is this enough for now?
Should we also already prepare a Dockerfile here or can we leave it for #401 ?

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,8 @@ GIT_TAG?=$(shell git describe --abbrev=0 --tags 2>/dev/null || echo "v0.0.0" )
 TAG?=${GIT_TAG}-${GIT_COMMIT_SHORT}
 REPO?=quay.io/costoolkit/elemental-operator-ci
 REPO_REGISTER?=quay.io/costoolkit/elemental-register-ci
+TAG_SEEDIMAGE?=${TAG}
+REPO_SEEDIMAGE?=quay.io/costoolkit/seedimage-builder
```
[the remark this answers is not in the document]
Comment (juadk) [reply Makefile:8]: better switching to `seedimage-builder-ci` for consistency yes

File: .obs/chartfile/values.yaml
Comment (davidcassany) [reply .obs/chartfile/values.yaml:1]: Just leave it for #401, I think with the current status it should not break the CI. It wouldn't be the end of the world if Dev is broken for a day or two, but better if we don't break it, so we don't have to hurry 😉

File: .obs/chartfile/values.yaml [hunk 2]
```diff
@@ -4,6 +4,10 @@ image:
   tag: "%VERSION%"
   imagePullPolicy: IfNotPresent
 
+seedImage:
+  repository: "quay.io/costoolkit/seedimage-builder"
+  tag: "latest"
+
```
Comment (davidcassany) [inline .obs/chartfile/values.yaml:10]: Yes I think this should be enough

**Review by davidcassany — APPROVED** [review]
Nice, I think this should be enough keep CI on Dev alive meanwhile we prepare #401

**Pull request merged**